File: bee2344.py
def main():
    nota = int(input())
    conceito = "E"
    if nota > 85: conceito = "A"
    elif nota > 60: conceito = "B"
    elif nota > 35: conceito = "C"
    elif nota > 0: conceito = "D"
    print(conceito)

main()

File: bee_2344.c
#include <stdio.h>

int main() {
    int nota;
    scanf("%d", &nota);

    char conceito = 'E';
    if (nota > 85) {
        conceito = 'A';
    } else if (nota > 60) {
        conceito = 'B';
    } else if (nota > 35) {
        conceito = 'C';
    } else if (nota > 0) {
        conceito = 'D';
    }
    printf("%c\n", conceito);
    
    return 0;    
}

